Jenkinsfile

```diff
@@ -58,12 +58,12 @@ def isPr() {
             stage('Prepare build(gcc)') {
             parallel (
                 '32bit' : {
-                    sh 'LDFLAGS=" -L/usr/icu32/lib/  -Wl,-rpath=/usr/icu32/lib/" cmake  -H./ -Bbuild/out_linux -DESCARGOT_HOST=linux -DESCARGOT_ARCH=x86 -DESCARGOT_MODE=debug -DESCARGOT_OUTPUT=shell_test -GNinja'
+                    sh 'LDFLAGS=" -L/usr/icu32/lib/  -Wl,-rpath=/usr/icu32/lib/" cmake  -H./ -Bbuild/out_linux -DESCARGOT_HOST=linux -DESCARGOT_ARCH=x86 -DESCARGOT_MODE=debug -DESCARGOT_DEBUGGER=1 -DESCARGOT_OUTPUT=shell_test -GNinja'
                     sh 'LDFLAGS=" -L/usr/icu32/lib/  -Wl,-rpath=/usr/icu32/lib/" cmake  -H./ -Bbuild/out_linux_release -DESCARGOT_HOST=linux -DESCARGOT_ARCH=x86 -DESCARGOT_MODE=release -DESCARGOT_OUTPUT=shell_test -GNinja'
                     sh 'gcc -shared -m32 -fPIC -o backtrace-hooking-32.so tools/test/test262/backtrace-hooking.c'
                 },
                 '64bit' : {
-                    sh 'cmake  -H./ -Bbuild/out_linux64 -DESCARGOT_HOST=linux -DESCARGOT_ARCH=x64 -DESCARGOT_MODE=debug -DESCARGOT_OUTPUT=shell_test -GNinja'
+                    sh 'cmake  -H./ -Bbuild/out_linux64 -DESCARGOT_HOST=linux -DESCARGOT_ARCH=x64 -DESCARGOT_MODE=debug -DESCARGOT_DEBUGGER=1 -DESCARGOT_OUTPUT=shell_test -GNinja'
```

Comment:
Shouldn't we create another binary rather than changing an existing one?

Reply:
I didn't want to add new binary, since it doesn't influence the other test-cases.

Reply:
Currently CI(Jenkins) runs `debug binary` for modifiedVendorTest, regression-tests, new-es, intl, sunspider-js and test262 TCs.
These tests take quite a short time, but test262 case might be affected by running `debugger-enabled binary` since test262 is one of the most time-consuming TCs.

Reply:
I'm not sure but if the total time is significantly increased it would be better to run `debug-only binary` for test262 case. What do you think about it?

Reply:
I added a debugger build to Jenkins. The only question have do we want to have both a 32-bit and a 64-bit debugger build or it is a 64-bit build enough?

Reply:
ATM 64-bit build would be enough. We may add 32-bit build later if it's really necessary. Thanks.